web/client/components/map/leaflet/plugins/WFSLayer.jsx

```diff
@@ -6,37 +6,36 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {isNil} from 'lodash';
+import { isNil, isEqual } from 'lodash';
 import L from 'leaflet';
+import { colorToRgbaStr } from '../../../../utils/ColorUtils';
 
 import CoordinatesUtils from '../../../../utils/CoordinatesUtils';
 import Layers from '../../../../utils/leaflet/Layers';
 import { optionsToVendorParams } from '../../../../utils/VendorParamsUtils';
 
 import { getFeature } from '../../../../api/WFS';
 import { needsReload } from '../../../../utils/WFSLayerUtils';
-const defaultStyle = {
-    radius: 5,
-    color: "red",
-    weight: 1,
-    opacity: 1,
-    fillOpacity: 1
-};
+
 
 const loadFeatures = (layer, options) => {
+    layer.fireEvent('loading');
     const params = optionsToVendorParams(options);
     const onError = () => {
+        layer.fireEvent('loadError');
         // // TODO: notify error
     };
     return getFeature(options.url, options.name, {
         // bbox: extent.join(',') + ',' + proj,
         outputFormat: "application/json",
-        srsname: CoordinatesUtils.normalizeSRS(options.srs || 'EPSG:3857', options.allowedSRS),
+        maxFeatures: 1000,
+        srsname: CoordinatesUtils.normalizeSRS( 'EPSG:4326'),
```

Comment:
What happens if EPSG:4326 is not supported on the server?

Reply:
Probably an error in the layer (not tested). 
I though for leaflet it was a minimal requirement. Do you think it is not?

Reply:
The problem is not leaflet here, it is the server. What if I have a WFS service that only supports EPSG:3857 but not EPSG:4326?

Reply:
I said what happens, an error notified on the layer, but I didn't get what you suggest. 
- Do you want I handle the specific error, maybe with a clear message ?
- Also the opposite is technically possible, support 4326 and not 3857. 
    - Do you think we should handle something like "allowedSRS" from getCapabilities (catalog) to the layer? in this case it should be handled also by the openlayers implementation.
    - Should we assume that 3857 is supported 
    - should we do a getCapabilities to WFS to get allowedSRS (if not present)

I think this minimal implementation for leaflet was sufficient. Can you be more explicit about what you think is wrong or can improved?

Reply:
What I would expect is to use the same srs of the current map, so that we avoid client side reprojection if not needed.

Reply:
For this reason I asked 4326, because leaflet works in 4326 coordinates internally, so I don't have to do any reprojection.
I though asking features in 3857 (the only possible projection of the map) and convert in 4326 was a unuseful complication (if we don't handle allowedSRS).
What do you think?

Reply:
ok, that makes sense, let's add allowedSRS use for a future improvement